Automattic/wp-calypso
# Performance Profiler: adds social media share.

Fixes https://github.com/Automattic/dotcom-forge/issues/8822

## Proposed Changes

* Adds sharing popover
* Adds twitter, facebook, linkedin, tumblr services
* Uses `share_link` from the backend

## Testing Instructions



* Visit `http://calypso.localhost:3000/speed-test-tool?url=https%3A%2F%2Fblecyclingclub.gr%2F&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NzAyOH0.bKYiVrIqTJtq0QP2eeZCsP8wRw1-e6sCAxVDsn89QCg&tab=mobile`
* Click Share Results
* You should land in twitter with a default prompt
* Repeat for facebook, linkedin

![CleanShot 2024-09-10 at 15 15 07](https://github.com/user-attachments/assets/f28e6724-9ab8-4996-be2c-fd66bdd6478c)



## Pre-merge Checklist
